internal/services/containers/container_registry_token_password_resource_test.go

```diff
@@ -56,7 +56,7 @@ func TestAccContainerRegistryTokenPassword_update(t *testing.T) {
 	data := acceptance.BuildTestData(t, "azurerm_container_registry_token_password", "test")
 	r := ContainerRegistryTokenPasswordResource{Expiry: time.Now().Add(time.Hour)}
 
-	data.ResourceTest(t, r, []acceptance.TestStep{
+	data.ResourceTestIgnoreRecreate(t, r, []acceptance.TestStep{
```

Comment:
These tests are called `_update` so presumably they're checking that certain properties are updating correctly on the resource and not triggering a ForceNew, which means that this isn't a proper fix for the test and is masking an underlying issue.

Reply:
The reason can be found at: https://github.com/hashicorp/terraform-provider-azurerm/blob/cd57967c0e377f6e70adc2bac576ce18b51a638e/internal/services/containers/container_registry_token_password_resource.go#L59-L66

Due to this bug, we are not able to *update* it, but can only recreate a new token password. This doesn't make a big difference for this resource.

Reply:
Agreed. I skip these two test cases now until we migrated to fw.